packages/container/__tests__/index.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import renderer from 'react-test-renderer'
+import { ThemeProvider } from '@hixme-ui/theme'
+import Container from '../src/index.js'
+
+test('Animate', () => {
```

Comment:
Should probably say 'Container'. I've made a more generic version of this in a couple of other components (`/table/__tests__/index.test.js`, for example) that's a bit easier to just copy and paste.